src/pages/dev/general-message-passing/overview.mdx

```diff
@@ -20,15 +20,95 @@ With GMP you can:
2.049 150.18 L 213.577 150.18 Q 214.285 150.18 214.688 150.048 Q 215.091 149.917 215.33 149.677 Q 215.667 149.34 215.855 148.772 Q 216.043 148.203 216.043 147.392 Q 216.043 146.269 215.674 145.666 Q 215.306 145.063 214.778 144.858 Q 214.397 144.712 213.553 144.712 L 212.049 144.712 Z M 221.576 150.385 Q 221.088 150.8 220.636 150.971 Q 220.185 151.142 219.667 151.142 Q 218.813 151.142 218.354 150.725 Q 217.895 150.307 217.895 149.658 Q 217.895 149.277 218.068 148.962 Q 218.241 148.647 218.522 148.457 Q 218.803 148.266 219.154 148.169 Q 219.413 148.1 219.936 148.037 Q 221 147.91 221.503 147.734 Q 221.508 147.553 221.508 147.504 Q 221.508 146.967 221.259 146.748 Q 220.922 146.45 220.258 146.45 Q 219.638 146.45 219.342 146.667 Q 219.047 146.884 218.905 147.436 L 218.046 147.319 Q 218.163 146.767 218.432 146.428 Q 218.7 146.088 219.208 145.905 Q 219.716 145.722 220.385 145.722 Q 221.049 145.722 221.464 145.879 Q 221.879 146.035 222.074 146.272 Q 222.27 146.508 222.348 146.87 Q 222.392 147.094 222.392 147.68 L 222.392 148.852 Q 222.392 150.078 222.448 150.402 Q 222.504 150.727 222.67 151.025 L 221.752 151.025 Q 221.615 150.752 221.576 150.385 Z M 221.503 148.422 Q 221.024 148.618 220.067 148.754 Q 219.525 148.833 219.301 148.93 Q 219.076 149.028 218.954 149.216 Q 218.832 149.404 218.832 149.633 Q 218.832 149.985 219.098 150.219 Q 219.364 150.454 219.877 150.454 Q 220.385 150.454 220.78 150.232 Q 221.176 150.009 221.361 149.624 Q 221.503 149.326 221.503 148.745 Z M 223.755 153.012 L 223.755 145.839 L 224.556 145.839 L 224.556 146.513 Q 224.839 146.118 225.195 145.92 Q 225.552 145.722 226.06 145.722 Q 226.724 145.722 227.232 146.064 Q 227.739 146.406 227.998 147.028 Q 228.257 147.651 228.257 148.393 Q 228.257 149.189 227.971 149.826 Q 227.686 150.463 227.141 150.803 Q 226.597 151.142 225.996 151.142 Q 225.557 151.142 225.208 150.957 Q 224.858 150.771 224.634 150.488 L 224.634 153.012 Z M 224.551 148.462 Q 224.551 149.462 224.956 149.941 Q 225.361 150.42 225.938 150.42 Q 226.523 150.42 226.941 149.924 Q 227.358 149.428 227.358 148.388 Q 227.358 147.397 226.951 146.904 Q 226.543 146.411 225.977 146.411 Q 225.415 146.411 224.983 146.936 Q 224.551 147.461 224.551 148.462 Z M 229.302 153.012 L 229.302 145.839 L 230.103 145.839 L 230.103 146.513 Q 230.386 146.118 230.742 145.92 Q 231.099 145.722 231.607 145.722 Q 232.271 145.722 232.778 146.064 Q 233.286 146.406 233.545 147.028 Q 233.804 147.651 233.804 148.393 Q 233.804 149.189 233.518 149.826 Q 233.233 150.463 232.688 150.803 Q 232.144 151.142 231.543 151.142 Q 231.104 151.142 230.755 150.957 Q 230.405 150.771 230.181 150.488 L 230.181 153.012 Z M 230.098 148.462 Q 230.098 149.462 230.503 149.941 Q 230.908 150.42 231.484 150.42 Q 232.07 150.42 232.488 149.924 Q 232.905 149.428 232.905 148.388 Q 232.905 147.397 232.498 146.904 Q 232.09 146.411 231.524 146.411 Q 230.962 146.411 230.53 146.936 Q 230.098 147.461 230.098 148.462 Z" style="fill: rgb(51, 51, 51); stroke: rgb(0, 0, 0); text-transform: capitalize;"/>
```

Comment:
Oh, I have tried compressing it, but it seems it can't go beyond that, and also, one of the reasons might be due to invisible elements (https://stackoverflow.com/questions/55224922/how-do-i-optimize-svg-paths) even though I filtered that severally.
@ffe9f8